ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -510,8 +510,16 @@ objects:
           echo "Gathering artifacts ..."
           mkdir -p /tmp/artifacts/pods /tmp/artifacts/nodes /tmp/artifacts/metrics /tmp/artifacts/bootstrap /tmp/artifacts/network
 
-          # Get nova logs
-          openstack console log show ${CLUSTER_NAME}-bootstrap &>/tmp/artifacts/bootstrap/nova.log
+          # Get bootstrap nova logs
+          # NOTE(shadower): The server names are in the form
+          # `$CLUSTER_NAME-fmszv-bootstrap`. I could not find how to
+          # get the middle part (`fmszv`) (it's not in the shell
+          # environment here) so we'll list all the servers and look
+          # the bootstrap name up that way.
+          local BOOTSTRAP_NAME=$(openstack server list --format value --column Name | grep "${CLUSTER_NAME}-.*-bootstrap")
+          if [ -n "$BOOTSTRAP_NAME" ]; then
+              openstack console log show ${CLUSTER_NAME}-bootstrap &>/tmp/artifacts/bootstrap/nova.log
```

Comment:
should this be '${BOOTSTRAP_NAME}` here?

Reply:
the behaviour is identical in this case

Reply:
Oh. By reading this, I was under the impression that

```
CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
BOOTSTRAP_NAME="${CLUSTER_NAME}-fmszv-bootstrap
```

so that `${CLUSTER_NAME}-bootstrap` would be

```
"${NAMESPACE}-${JOB_NAME_HASH}-bootstrap"
```

instead of

```
"${NAMESPACE}-${JOB_NAME_HASH}-fmszv-bootstrap"
```

Reply:
oh sorry I misread the line and what you meant. Yes, thanks!

Reply:
Fixed